README.md

```diff
@@ -85,9 +85,16 @@ Note that the website will automatically reload as you edit the code.
 
 ### Configuration
 
-Configuration options for the Argus frontend are located in `src/config.tsx`. All of these options can be set using environment variables when running the frontend under the Node server (or in Docker Compose). However, for production deployment, you would normally build the application and serve all the resulting static files using a regular web server, like Apache or Nginx; in this case, `config.tsx` cannot read server environment varibles, and should be hard coded instead.
+Default configuration options for the Argus frontend are located in `src/config.tsx`. All of these options can be set using **environment** variables when running the frontend under the Node server (or in Docker Compose), or by providing **configuration** variables in `runtime-config.json` file.
 
-These environment variables are available:
+#### Development environment
+Either provide **environment** variables when running the frontend under the Node server (or in Docker Compose), or add `runtime-config.json` file with the **configuration** variables to the `/public` folder.
+
+#### Production environment
+Serve `./runtime-config.json` file with the **configuration** variables. 
+
+#### Variables
+These **environment** variables are available:
```

Comment:
You should also mention somewhere that these environment variables can also be used as described here if using the production-oriented Docker image defined in `./docker/Dockerfile`.  This image will produce a `runtime-config.json` from the environment variables exported to the container when it starts.

Perhaps even link to the `docker/README.md` file (which I see I also need to update, since it is outdated with the changes I contributed to this PR.